Remove npm-check dependency

jzolnowski opened the pull request:
Fix the [Got allows a redirect to a UNIX socket](https://github.com/xlts-dev/angularjs-webpack-babel/security/dependabot/40) security issue

Commit:
chore(package): replace npm-check with npm-check-updates
- remove `npm check` due to
  - [got allows a redirect to a unix socket](https://github.com/xlts-dev/angularjs-webpack-babel/security/dependabot/40) issue
  - lack of regular maintenance
- add [npm-check-updates](https://github.com/raineorshine/npm-check-updates)[npm-check-updates](https://github.com/raineorshine/npm-check-updates)
- update 'update' npm script

gkalpak commented on package-lock.json:
We shouldn't be changing the lockfile version in this PR.

jzolnowski commented:
I updated the commit message, generated package-json without changing `lockfileVersion`, replaced `npm-check` with [npm-check-updates](https://github.com/raineorshine/npm-check-updates). At this point, it doesn't have any [vulnerabilities](https://security.snyk.io/package/npm/npm-check-updates)

gkalpak commented on package.json:
```diff
@@ -60,7 +60,7 @@
     "test:e2e:report": "npx playwright show-report",
     "test": "run-s lint test:unit:single test:e2e",
-    "update": "npm-check -u",
+    "update": "ncu -u",
```
If I am not mistaken, `ncu -u` only updates `package.json`, but it does not install the updated versions (and most importantly it does not update the lockfile). Should we manually install:

```suggestion
    "update": "ncu -u && npm install",
```

jzolnowski replied:
Updated

gkalpak reviewed (changes requested):
Thx for adding info and links in the commit message. However, it seems that the commit message header and body have been merged into one (which is not the correct format).

I would expected something like:

> chore(package): replace \`npm-check\` devDependency with \`npm-check-updates\`
>
> Replace \`npm-check\` devDependency with [npm-check-updates][1] due to the
> [got allows a redirect to a unix socket][2] security issue.
>
> [1]: https://github.com/raineorshine/npm-check-updates
> [2]: https://github.com/xlts-dev/angularjs-webpack-babel/security/dependabot/40

jzolnowski commented:
Commit message updated, ready for another round 😄

gkalpak reviewed (approved):
The changes lgtm :+1:

(The commit message body is in a funny format, but 🤷‍♂️ :grin:)
